pkg/pkgupdates/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<!--
+This file is part of Cockpit.
+
+Copyright (C) 2017 Red Hat, Inc.
+
+Cockpit is free software; you can redistribute it and/or modify it
+under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation; either version 2.1 of the License, or
+(at your option) any later version.
+
+Cockpit is distributed in the hope that it will be useful, but
+WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+-->
+<html>
+
+<head>
+  <title translate>Software Updates</title>
+  <meta charset="utf-8">
+
+  <link href="../base1/patternfly.css" type="text/css" rel="stylesheet">
+  <link href="../base1/cockpit.css" rel="stylesheet">
+  <link href="pkgupdates.css" rel="stylesheet">
+
+  <script src="../base1/jquery.js"></script>
```

Comment:
I do use it for setting D-Bus signal handlers with `$(myDbusProxy).on(...)`. The JSX already `require()s` it, but without this `<script>` here I get an Oops about "ReferenceError: jQuery is not defined".